Fixed #32076 -- Added async methods to BaseCache. (#13547)
django/django: docs/releases/4.0.txt

Cache
~~~~~

> * The new async API for ``django.core.cache.backends.base.BaseCache`` begins
>   process of making cache backends async-compatible. All new methods have the
>   ``a`` prefixed names, e.g. ``aadd()``, ``aget()``, ``aset()``,
>   ``aget_or_set()``, or ``adelete_many()``. The ``a`` prefix will be used in
>   the future for async variants of any methods.

Reviewer: `begins the process` but, maybe... 

```suggestion
* The new async API for ``django.core.cache.backends.base.BaseCache`` begins 
  the process of making cache backends async-compatible. The new async methods
  all have ``a`` prefixed names, e.g. ``aadd()``, ``aget()``, ``aset()``,
  ``aget_or_set()``, or ``adelete_many()``. 
  
  Going forward, the ``a`` prefix will be used for async variants of
  methods generally.
```

Author: @felixxm I'm 👍 with this.

>   the future for async variants of any methods.

Author: I'm confused are we still keeping the docs for 4.0? What happened to the revert on the 3.2 docs?

Reviewer 2: It's a new features so it will included in the Django 4.0. Previously I requested to revert adding and empty _"Cache"_ section to the Django 3.2. release notes.
